test/p3_tests.jl

```diff
@@ -177,20 +177,22 @@ function test_p3_shape_solver(FT)
                         # Convert λ to ensure it remains positive
                         x = log(λ_ex)
                         # Compute mass density based on input shape parameters
-                        q_calc = P3.q_gamma(p3, F_r, N, x, th)
-
-                        # Solve for shape parameters
-                        (λ, N₀) = P3.distribution_parameter_solver(
-                            p3,
-                            q_calc,
-                            N,
-                            ρ_r,
-                            F_r,
-                        )
-
-                        # Compare solved values with the input expected values
-                        TT.@test λ ≈ λ_ex rtol = ep
-                        TT.@test N₀ ≈ N₀_ex rtol = ep
+                        q_calc = N * P3.q_gamma(p3, F_r, x, μ_ex, th)
+
+                        if q_calc < FT(1)
```

Comment:
`q = 1 kg/kg` is still too large to be physical. Lets cut it off at about `0.05`. (50 grams per kilogram is still larger than what we would normally expect)

Do we gen enough testing points like that. And if yes, is the error on them smaller than eps=1?